.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+stale_process_union=( "docker-containerd-shim" "qemu-system-aarch64" )
+stale_docker_mount_point_union=( "/var/lib/docker/containers" "/var/lib/docker/overlay2" )
+stale_docker_dir="/var/lib/docker"
+stale_kata_dir="/var/lib/vc/sbs"
+
+lib_script="${GOPATH}/src/${tests_repo}/.ci/lib.sh"
+source "${lib_script}"
+
+kill_stale_process()
```

Comment:
We should probably leverage the existing functions in:

- https://github.com/kata-containers/tests/blob/master/metrics/lib/common.bash

They'll need to be moved to a `./lib/common.bash` though probably.

Reply:
yes, I will use the existing functions` kill_processes_before_start` to substitute much of the logic here. It dealt with most of the stale processes. The only left part will be the 'docker-containerd-shim' process which i will  use `pgrep` or `pkill` to eliminate them as @grahamwhaley suggested.
If you are considering to move referred files to `./lib/common.bash` lately, maybe I should wait for them landed firstly? or just ping me at that time, I will refine here accordingly. 😊